content/en/examples/pods/pod-with-affinity-anti-affinity.yaml

```diff
@@ -29,4 +30,4 @@ spec:
             - key-2
   containers:
   - name: with-node-affinity
-    image: k8s.gcr.io/pause:2.0
```

Comment:
@deRemo 👋🏻 This looks like an unnecessary change. For future PRs, please configure your editor not to enforce newlines.